mlir/test/lib/Dialect/Test/TestAttrDefs.td

```diff
@@ -354,9 +354,9 @@ def TestCustomFloatAttr : Test_Attr<"TestCustomFloat"> {
 
 def NestedPolynomialAttr : Test_Attr<"NestedPolynomialAttr"> {
   let mnemonic = "nested_polynomial";
-  let parameters = (ins Polynomial_IntPolynomialAttr:$poly);
+  let parameters = (ins OptionalParameter<"::mlir::polynomial::IntPolynomialAttr">:$poly);
```

Comment:
Aren't we losing coverage for the previous fix that introduced this attribute?

Reply:
Yes, I will add a separate test.

Reply:
Fixed in https://github.com/llvm/llvm-project/pull/96667/commits/7b59a7d2e6506e44debd4a212a35b8ede4817042